recipes/starlette-csrf/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: starlette-csrf
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/starlette-csrf/starlette_csrf-{{ version }}.tar.gz
+  sha256: 5f03b8c496840d612f1f7065aac7a209da084972f6e7844bcc5752c32ac54747
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
You don't need the `--no-deps --no-build-isolation` but it doesn't hurt. (You do need them when using multiple outputs though but that is more a bug in our toolchain than a real requirement.)